common/clickhouse-srv/src/types/column/fixed_string.rs

```diff
@@ -118,7 +118,7 @@ impl<K: ColumnType> ColumnData for FixedStringAdapter<K> {
             match self.column.at(index) {
                 ValueRef::String(_) => {
                     let string_ref = self.column.at(index).as_bytes().unwrap();
-                    buffer.extend(string_ref.as_ref());
+                    buffer.extend::<&[u8]>(string_ref.as_ref());
```

Comment:
This to fix the warning:
error[E0282]: type annotations needed
   --> common/clickhouse-srv/src/types/column/fixed_string.rs:121:28
    |
121 |                     buffer.extend(string_ref.as_ref());
    |                            ^^^^^^ cannot infer type for type parameter `A` declared on the trait `Extend`